src/theme/qml/ComboBox.qml

```diff
@@ -44,32 +44,64 @@ T.ComboBox {
     hoverEnabled: true
 
     delegate: MenuItem {
-        width: parent.width
+        width: parent.width - 5
```

Comment:
I don't like using fixed values, instead use something like `2 * theme.frameWidth` so it makes sense.